documentation/docs/release-notes/3.2.0.md

```diff
@@ -0,0 +1,159 @@
+
+### Secure external ClickHouse connections 
+PMM now supports connecting to external ClickHouse servers using authenticated credentials. 
+
+You can now set the `PMM_CLICKHOUSE_USER` and `PMM_CLICKHOUSE_PASSWORD` environment variables when deploying PMM Server to enable access to password-protected ClickHouse instances, such as those managed by Altinity’s operator. 
+
+This means that previous authentication errors are resolved, you no longer need to embed credentials in connection strings, which makes your setup more secure.
+
+### Feature Compatibility Version (FCV) panel in MongoDB dashboards
+PMM now shows the current MongoDB Feature Compatibility Version (FCV) in the  **MongoDB Replica Set** and **MongoDB Sharded Cluster Summary** dashboards. 
+
+This panel helps you quickly see if your cluster is running the expected FCV—especially after upgrades, when the FCV might not match the MongoDB version yet.
+
+![FCV panels](../images/FCV_panel.png)
+
+### New PBM Details dashboard for MongoDB backups
+
+PMM now includes a new **MongoDB PBM Details** dashboard that enables you to monitor MongoDB backups directly from PMM. This new dashboard is powered by the PBM collector and offers a unified, real-time view of all your backup activity across replica sets and sharded clusters. 
+
+It displays key information such as backup status, configuration, size, duration, PITR status, and details of the most recent successful backup—all in one place.
+If you're already using Percona Backup for MongoDB (PBM) with PMM, this integration lets you track backup operations without switching between tools.
+
+To get started, open the [PBM Details dashboard](../reference/dashboards/dashboard-mongodb-PBM-details.md) and see how it works with your MongoDB environments.
+
+![PBM dashboard](../images/PBM_Dashboard.png)
+
+Future releases will improve the **Backup history** panel to better display certain error states and special conditions (like *stuck* or *incompatible* backups) that are currently only visible when using `PBM's pbm status` command directly.
+
+### Improved Query Analytics filter performance
+We've optimized QAN filter loading performance to reduce the number of processed rows by up to 95% in large environments. This means that the filters on the **PMM Query Analytics** page page now load faster, which makes the interface more responsive. 
+
+![QAN filters](../images/QAN_filters.png)
+
+### Nomad integration for future extensibility
+
+We've integrated [HashiCorp Nomad](https://developer.hashicorp.com/nomad) into PMM to lay the groundwork for future feature development. This workload orchestrator will initially manage monitoring agents to improve service delivery for Percona customers.
+
+For advanced users, the Nomad API (internal port `4646`) is accessible via the standard PMM HTTPS port using the `/nomad` prefix and requires Administrator privileges.
```

Comment:
I think you mean URL and not prefix (i.e. https://\<pmm-server\>/nomad) but if I'm misinterpreting that then it should be reworked because I'm not sure what else you meant...